src/mono/browser/build/BrowserWasmApp.targets

```diff
@@ -28,7 +28,7 @@
       _WasmGenerateRunV8Script;
     </WasmGenerateAppBundleDependsOn>
 
-    <WasmEnableJsInteropByValue Condition="'$(WasmEnableJsInteropByValue)' == '' and ( '$(WasmEnableThreads)' == 'true' or '$(MonoWasmBuildVariant)' == 'multithread' )">true</WasmEnableJsInteropByValue>
+    <WasmEnableJsInteropByValue Condition="'$(WasmEnableJsInteropByValue)' == '' and ( '$(WasmEnableThreads)' == 'true' or '$(WasmEnableThreads)' == 'true' )">true</WasmEnableJsInteropByValue>
```

Comment:
there is teh same condition twice now. And I think it should be `FeatureWasmManagedThreads` now

Reply:
It should be `FeatureWasmManagedThreads` only if the original value was `FeatureWasmThreads` because that was the rename process.